client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-dotcom-patterns.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery, UseQueryOptions } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+import type { Pattern } from '../types';
+
+const useDotcomPatterns = (
+	lang?: string,
+	queryOptions: UseQueryOptions< any, unknown, Pattern[] > = {}
+): Pattern[] => {
+	const { data } = useQuery< any, unknown, Pattern[] >(
+		[ lang, 'patterns' ],
+		() => {
+			return wpcomRequest( {
+				path: `/ptk/patterns/es`,
```

Comment:
hmmm...why do we always fetch `es` patterns 😂

Reply:
Oops. That was a test. Reverted. 

There is an issue with the `lang` param. Please see this follow-up [issue](https://github.com/Automattic/wp-calypso/issues/75421).